web/src/features/moreCast2/components/TabbedDataGrid.tsx

```diff
@@ -397,7 +393,12 @@ const TabbedDataGrid = ({ morecast2Rows, fromTo, setFromTo }: TabbedDataGridProp
   // row has no actual value for the weather parameter of interest)
   const handleCellDoubleClick = (params: GridCellParams) => {
     const headerName = params.colDef.headerName as WeatherDeterminateType
-    if (!headerName || headerName === WeatherDeterminate.ACTUAL || headerName === WeatherDeterminate.FORECAST) {
+    if (
+      !headerName ||
+      headerName === WeatherDeterminate.ACTUAL ||
+      headerName === WeatherDeterminate.FORECAST ||
+      params.field.indexOf('grass') > -1
```

Comment:
What does an index of -1 represent here?

Reply:
Sorry, I went old school on you there. string.indexOf will return -1 if the specified string is not included in `params.field`. The condition is analagous to the much more readable `params.field.includes('grass')`.

Reply:
Switch to use `.includes()`